FormHoneypotPlugin.inc.php

```diff
@@ -278,15 +326,26 @@ function getTemplatePath($inCore = false) {
 	 */
 	function handleTemplateDisplay($hookName, $args) {
 		
-		$templateMgr = $args[0];
+		// Could get this from $args[0] in this case but the result is identical
+		// and this is less likely to change
+		$templateMgr = TemplateManager::getManager();
+		
 		$template = $args[1];
+		$contextID = (!is_null($templateMgr->get_template_vars('currentJournal')) ? $templateMgr->get_template_vars('currentJournal')->getId() : CONTEXT_SITE);
```

Comment:
This will throw a deprecated warning in 3.1.2 and later.

https://github.com/pkp/pkp-lib/blob/6db2aab462807c62c36e2c0791c53bd9794fcc87/classes/template/PKPTemplateManager.inc.php#L1772-L1779

Reply:
Fixed this and the others in 31f0fce.